### PR TITLE
DTRUNETWO-301-Added getBankAccountStatusTransition Method in V3

### PR DESCRIPTION
**Pull request by jkurra-hw**

### CODE REVIEW

**Review by arao6 — APPROVED** [review]

**Review by vkorotchenko — APPROVED** [review]

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java [hunk 1]
```diff
@@ -4979,6 +4982,72 @@ public void testCreateBankAccountStatusTransition_successful() throws Exception
...
+        HyperwalletApiClient mockApiClient = createAndInjectHyperwalletApiClientMock(client);
+
+        Mockito.when(mockApiClient.get(Mockito.anyString(), Mockito.any(Class.class))).thenReturn(transitionResponse);
+
+        HyperwalletStatusTransition resp =
+                client.getBankAccountStatusTransition("test-user-token", "test-bank-account-token", "test-status-transition-token");
+        assertThat(resp, is(equalTo(transitionResponse)));
+        Mockito.verify(mockApiClient)
+                .get("https://api.sandbox.hyperwallet.com/rest/v3/users/test-user-token/bank-accounts/test-bank-account-token/status-transitions"
+                                + "/test-status-transition-token",
```
Comment (Rrathinasabapath) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java:5047]: Please make lines 5047 and 5048 into one line and format it.

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java [hunk 2]
```diff
@@ -4969,7 +4969,10 @@ public void testCreateBankAccountStatusTransition_successful() throws Exception
         assertThat(resp, is(equalTo(transitionResponse)));
 
         ArgumentCaptor<HyperwalletStatusTransition> argument = ArgumentCaptor.forClass(HyperwalletStatusTransition.class);
-        Mockito.verify(mockApiClient).post(Mockito.eq("https://api.sandbox.hyperwallet.com/rest/v3/users/test-user-token/bank-accounts/test-bank-account-token/status-transitions"), argument.capture(), Mockito.eq(transition.getClass()));
+        Mockito.verify(mockApiClient).post(Mockito
+                        .eq("https://api.sandbox.hyperwallet.com/rest/v3/users/test-user-token/bank-accounts/test-bank-account-token/status"
+                        + "-transitions"),
```
Comment (Rrathinasabapath) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java:4974]: Please make lines 4974 and 4975 into one line and format it.

**Review by Rrathinasabapath — CHANGES_REQUESTED** [review]
Pls make the minor formatting changes and resubmit the PR.

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java (hunk 1 above)
Comment (jkurra-hw) [reply src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java]: Code formatting is done . Thanks @Rrathinasabapath

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java (hunk 2 above)
Comment (jkurra-hw) [reply src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java]: Code formatting is done . Thanks @Rrathinasabapath

**Review by Rrathinasabapath — APPROVED** [review]
Approved. Ready to be reviewed by others.

**Pull request merged**